cogs/deadline.py

```diff
@@ -473,23 +512,89 @@ async def clearallreminders_error(self, ctx, error):
+    @tasks.loop(hours=24)
+    async def send_reminders_day(self):
+        channel = discord.utils.get(self.bot.get_all_channels(), name="reminders")
+        if channel:
+            reminders = db.query("SELECT course, homework, due_date "
+                "FROM reminders "
+                "WHERE due_date::date = now()::date")
+            for course,homework,due_date in reminders:
+                difference = due_date - datetime.now(timezone.utc)
+                await channel.send(f"{homework} for {course} is due in {(difference.seconds//3600)} hours")
+
+    # -----------------------------------------------------------------------------------------------------
+    #    Function: bofore(self)
+    #    Description: runs once per day and waits until 8:00 AM EST to send reminders via the send
+    #       reminders day function
+    #    Inputs:
+    #    - self: used to access parameters passed to the class through the constructor
+    # -----------------------------------------------------------------------------------------------------
+    @send_reminders_day.before_loop
+    async def before(self):
+        WHEN = time(13, 0, 0) # 8:00 AM eastern
+        now = datetime.utcnow()
+        target_time = datetime.combine(now.date(), WHEN)
+        seconds_until_target = (target_time - now).total_seconds()
+        if seconds_until_target < 0:
+            target_time = datetime.combine(now.date()+timedelta(days=1), WHEN)
+            seconds_until_target = (target_time - now).total_seconds()
+        await asyncio.sleep(seconds_until_target)
+
+    # -----------------------------------------------------------------------------------------------------
+    #    Function: send_reminders_hour(self)
+    #    Description: task that runs once per hours and sends a reminders for assignments due
+    #    Inputs:
+    #    - self: used to access parameters passed to the class through the constructor
+    # -----------------------------------------------------------------------------------------------------
+    @tasks.loop(hours=1)
+    async def send_reminders_hour(self):
+        channel = discord.utils.get(self.bot.get_all_channels(), name="reminders")
+        if channel:
+            reminders = db.query("SELECT course, homework, due_date "
+                "FROM reminders "
+                "WHERE due_date::date = now()::date")
+            for course,homework,due_date in reminders:
+                difference = due_date - datetime.now(timezone.utc)
+                if difference.seconds//3600 == 0:
+                    await channel.send(f"{homework} for {course} is due within the hour")
 
 # -------------------------------------
 # add the file to the bot's cog system
 # -------------------------------------
 def setup(bot):
     n = Deadline(bot)
-    loop = asyncio.get_event_loop()
-    # TODO
-    loop.create_task(n.delete_old_reminders())
+    n.send_reminders_day.start()
+    n.send_reminders_hour.start()
```

Comment:
Pylint fix!
Just add # pylint: disable=no-member at the end of the lines. It's a common false positive.

```suggestion
    n.send_reminders_day.start() # pylint: disable=no-member
    n.send_reminders_hour.start() # pylint: disable=no-member
```